README.md

```diff
@@ -1 +1,186 @@
+  </a>
+
+  <h3 align="center">Papercuts Bookstore</h3>
+
+  <p align="center">
+    OUR MISSION: An application to connect your community through reading and sharing books. 
+    <br />
+    <a href="https://github.com/hratx-blue-ocean/Papercuts"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/hratx-blue-ocean/Papercuts">View Demo</a>
+    ·
+    <a href="https://github.com/hratx-blue-ocean/Papercuts/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/hratx-blue-ocean/Papercuts/issues">Request Feature</a>
+  </p>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgements">Acknowledgements</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+#### Applicatin login with proper authentication
+
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
+
+Project Description
+
+Key Features
+
+- User authentication
+- Display the book of the month
+- Explore trending books from the store
+- View user libary and purchased book
+- Explore user libary and
+- Connect and add friends
+- Create bookclub and bookclub event
+- Email notification for bookclub event
+
+### Built With
+
+This section should list any major frameworks that you built your project using. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+
+- [Bootstrap](https://getbootstrap.com)
+- [React/Hooks](https://reactjs.org/)
+- [React-bootstrap](https://react-bootstrap.github.io/)
+- [MongoDB](https://www.mongodb.com/3)
+- [Node.js](https://nodejs.org/en/)
+- [Express.js](http://expressjs.com/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+
+- npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+
+1. Get a free API Key at [https://example.com](https://example.com)
+2. Clone the repo
+   ```sh
+   git clone https://github.com/hratx-blue-ocean/Papercuts.git
+   ```
+3. Install NPM packages
+   ```sh
+   npm install
+   ```
+4. Enter your API in `config.js`
```

Comment:
Need to tell them where. Is it in the client folder? If so where in the client

Reply:
this is just a template to start with matt, You guys need to help me writing other parts